src/stdlib/thunk_list.ts

```diff
@@ -0,0 +1,150 @@
+import { stringify } from '../utils/stringify'
+import { dethunk } from '../interpreter/thunk'
+
+// list.ts: Supporting lists in the Scheme style, using pairs made
+//          up of two-element JavaScript array (vector)
+// Author: Martin Henz
+// Translated to TypeScript by Evan Sebastian
+export type Pair<H, T> = [H, T]
+export type List = null | NonEmptyList
+interface NonEmptyList extends Pair<any, any> {}
+
+// array test works differently for Rhino and
+// the Firefox environment (especially Web Console)
+function array_test(x: any) {
+  if (Array.isArray === undefined) {
+    return x instanceof Array
+  } else {
+    return Array.isArray(x)
+  }
+}
+
+const descriptor = Object.create(null); // no inherited properties
+descriptor.value = true;
```

Comment:
`const descriptor = true;`

Reply:
Descriptor has to be a object or the code can't run, if is a boolean value then the code can't run